frontend/components/Buttons/PlayButton.vue

```diff
@@ -76,22 +77,21 @@ export default Vue.extend({
     };
   },
   computed: {
-    ...mapState('playbackManager', ['status'])
+    ...mapStores(playbackManagerStore)
   },
   watch: {
-    status(): void {
-      if (this.status === PlaybackStatus.Playing) {
+    'playbackManager.status'(): void {
```

Comment:
This kind of syntax allows us to watch for changes for a specific property in an object. So, if we have:

```js
mainKey: {
    key1: 'string',
    key2: 3,
    key3: ['hello', 1]
}
```

If we set the watcher as:

```js
mainKey(): void {
    ...
}
```

The handler will run if any of the **values** (not keys, see [here](https://github.com/jellyfin/jellyfin-vue/pull/1746#discussion_r862503703)) changes. If we declare the watcher as:

```js
'mainKey.key2'(): void {
    ...
}
```

We will be watching **value** changes just in ``key2``.

Keys can also be accessed through string, both these statements are equal:

```js
mainKey.key1
mainKey['key1']
```

This is why the handler is between quotes, because that syntax is also valid to access values in an object. Doing it without quotes **makes VSCode turn everything red (although, to be honest, I didn't bother checking if compilation was still successful or not, because in that case it's a problem we can fix by revisiting our linting config)**.

I can recheck this, but perhaps it's not a linter issue and **dots are not supported in a function declaration** in the ECMAScript specification

Reply:
Thanks! Seems you're right https://v2.vuejs.org/v2/api/#watch